README.md

```diff
@@ -84,6 +59,235 @@ on the site and the users billing Address is set to "Austria", GoCommerce will v
+
+## Configuration
+
+You may configure GoCommerce using either a configuration file named `.env`,
+environment variables, or a combination of both. Environment variables are prefixed with `GOCOMMERCE_`, and will always have precedence over values provided via file.
+
+### Top-Level
+
+```
+GOCOMMERCE_SITE_URL=https://example.netlify.com/
+```
+
+`SITE_URL` - `string` **required**
+
+The base URL your site is located at.
+
+`OPERATOR_TOKEN` - `string` *Multi-instance mode only*
+
+The shared secret with an operator (usually Netlify) for this microservice. Used to verify requests have been proxied through the operator and
+the payload values can be trusted.
+
+### API
+
+```
+GOCOMMERCE_API_HOST=localhost
+PORT=9999
+```
+
+`API_HOST` - `string`
+
+Hostname to listen on.
+
+`PORT` (no prefix) / `API_PORT` - `number`
+
+Port number to listen on. Defaults to `8080`.
+
+`API_ENDPOINT` - `string` *Multi-instance mode only*
+
+Controls what endpoint Netlify can access this API on.
+
+### Database
+
+```
+GOCOMMERCE_DB_DRIVER=sqlite3
+DATABASE_URL=gotrue.db
+```
+
+`DB_DRIVER` - `string` **required**
+
+Chooses what dialect of database you want. Choose from `sqlite3`, `mysql`, or `postgres`.
+
+`DATABASE_URL` (no prefix) / `DB_DATABASE_URL` - `string` **required**
+
+Connection string for the database. See the [gorm examples](https://github.com/jinzhu/gorm/blob/gh-pages/documents/database.md) for more details.
+
+`DB_NAMESPACE` - `string`
+
+Adds a prefix to all table names.
+
+`DB_AUTOMIGRATE` - `bool`
+
+If enabled, creates missing tables and columns upon startup.
+
+### Logging
+
+```
+LOG_LEVEL=debug
+```
+
+`LOG_LEVEL` - `string`
+
+Controls what log levels are output. Choose from `panic`, `fatal`, `error`, `warn`, `info`, or `debug`. Defaults to `info`.
+
+`LOG_FILE` - `string`
+
+If you wish logs to be written to a file, set `log_file` to a valid file path.
+
+### Payment
+
+#### Stripe
+
+`PAYMENT_STRIPE_ENABLED` - `bool`
+
+Whether Stripe is enabled as a payment provider or not.
+
+`PAYMENT_STRIPE_SECRET_KEY` - `string`
+
+The Stripe [secret key](https://stripe.com/docs/api#authentication) used when authenticating with the Stripe API.
+
+#### PayPal
+
+`PAYMENT_PAYPAL_ENABLED` - `bool`
+
+Whether PayPal is enabled as a payment provider or not.
+
+`PAYMENT_PAYPAL_CLIENT_ID` - `string`
+`PAYMENT_PAYPAL_SECRET` - `string`
+
+The OAuth credentials PayPal issued to you. GoCommerce will use them to [obtain an access token](https://developer.paypal.com/docs/api/overview/#authentication-and-authorization).
+
+`PAYMENT_PAYPAL_ENV` - `string`
+
+The PayPal environment to use. Choose from `production` or `sandbox`.
+
+### Downloads
+
+`DOWNLOADS_PROVIDER` - `string`
+
+The provider to use for downloads. Choose from `netlify` or ``.
+
+`DOWNLOADS_NETLIFY_TOKEN` - `string`
+
+The authentication bearer token used to access the Netlify downloads API.
+
+### Coupons
+
+`COUPONS_URL` - `string`
+
+A URL that contains all the coupon information in JSON.
```

Comment:
might be worth specifying if this is a relative path to the site URL or a complete URL path

Reply:
IMHO a URL is absolute. If it was relative, it would be called a path and mention what it was relative to.

Reply:
hmm ok, I for some reason thought that it was relative to the site, but it has been a long time since I looked. Good clarification, no change necessary.